### PR TITLE
Upgrade clsx to v2

### PR DESCRIPTION
**Pull request by MohammedFaragallah**

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -9205,20 +9205,20 @@ __metadata:
...
   checksum: c09c00ad14f638366ca814097e6cab533dfa1972a358da5b557be487168acbb25b4c1395e89ffa842a8a61ba87a462d2b4885bc9d4f8410b598f3cb339599cdb
   languageName: node
   linkType: hard
 
+"clsx@npm:^2.1.1":
+  version: 2.1.1
+  resolution: "clsx@npm:2.1.1"
+  checksum: c4c8eb865f8c82baab07e71bfa8897c73454881c4f99d6bc81585aecd7c441746c1399d08363dc096c550cceaf97bd4ce1e8854e1771e9998d9f94c4fe075839
+  languageName: node
+  linkType: hard
```
Comment (djhi) [inline yarn.lock]:
```suggestion
"clsx@npm:^2.1.0, clsx@npm:^2.1.1":
  version: 2.1.1
  resolution: "clsx@npm:2.1.1"
  checksum: c4c8eb865f8c82baab07e71bfa8897c73454881c4f99d6bc81585aecd7c441746c1399d08363dc096c550cceaf97bd4ce1e8854e1771e9998d9f94c4fe075839
  languageName: node
  linkType: hard
```

**Review by djhi — CHANGES_REQUESTED** [review]
Thanks! I can't push on your fork though and there's a duplicate dependency

File: yarn.lock (hunk 1 above)
Comment (MohammedFaragallah) [reply yarn.lock]: Thanks for your review, but I'm not sure what to do now my fork has your changes already.

Comment (djhi) [reply yarn.lock]: the git cli told me my commit was rejected but actually it worked... Nothing to do on your side :+1:

**Review by djhi — APPROVED** [review]

**Pull request merged**